frontend/reports/poverty-line/poverty-line-results.component.tsx

```diff
@@ -3,6 +3,7 @@ import { useReportsApi } from "../shared/use-reports-api";
 import BasicTableReport from "../shared/basic-table-report.component";
 import { formatPercentage } from "../shared/report.helpers";
 import { sum, values } from "lodash-es";
+import { CsvOptions } from "../../util/csv-utils";
```

Comment:
It seems you may have mixed branches here - this code was in #800 but is also in this pull request. Ideally each pull request would be independent and not have the same code in them.

There are some ways to fix it with git, but the easiest might be to address the feedback I left in #800 and get that merged. After it's merged, you merge master into issue-789-4 and this pull request will only have the two new reports rather than the two reports from #800